spec/system/admin/users_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+describe 'Users', :js do
+  let(:url) { '/admin/users' }
+  let(:downloaded_csv) { 'users.csv' }
+
+  it_behaves_like 'a page restricted to super admins'
+  it_behaves_like 'a CSV export'
+
```

Comment:
include the shared examples for an active scaffold page

Reply:
Added.